### PR TITLE
Origin/editorialreview cloudconsole

### PR DESCRIPTION
**Pull request by christinaausley**
Editorial review of all Cloud Console docs for grammar, punctuation, and organization. A few comments made inline. Looking forward to your feedback!

### CODE REVIEW

File: docs/product-manuals/cloud-console/manage-clusters/create-cluster.md
Comment (urbanisierung) [inline docs/product-manuals/cloud-console/manage-clusters/create-cluster.md:12]: @christinaausley The snippet above is used to include another Markdown file into this one. On this page everything from the `create-cluster-include.md` file will be included. So nothing else to do here :)

File: docs/product-manuals/cloud-console/manage-clusters/manage-api-clients.md
Comment (urbanisierung) [inline docs/product-manuals/cloud-console/manage-clusters/manage-api-clients.md:46]: Oops ;) We can just cut off before the comma: `A new token can be requested at this address.`

File: docs/product-manuals/cloud-console/manage-organization/manage-users.md
Comment (urbanisierung) [inline docs/product-manuals/cloud-console/manage-organization/manage-users.md:36]: Exactly, your proposal is much better: `People who do not yet have a Camunda Cloud account can also be invited to an organization.`

File: docs/product-manuals/cloud-console/troubleshooting/common-pitfalls.md
Comment (urbanisierung) [inline docs/product-manuals/cloud-console/troubleshooting/common-pitfalls.md:21]: The cluster status can be observed in both views: the cluster list and the cluster details. A user can see if a cluster is healthy by navigating to the cluster list, so there is no need to go to the cluster details. But: the cluster details give the user a more fine grained view of the status over all products (zeebe, operate, tasklist, optimize).

File: docs/product-manuals/cloud-console/troubleshooting/feedback-and-support.md
Comment (urbanisierung) [inline docs/product-manuals/cloud-console/troubleshooting/feedback-and-support.md:8]: In the beginning it was a built-in form. Now we just link to the Forum: https://forum.camunda.io/ - so we can use the forum link here.

File: docs/product-manuals/cloud-console/manage-clusters/create-cluster.md
Comment (christinaausley) [reply docs/product-manuals/cloud-console/manage-clusters/create-cluster.md:12]: Great, thanks for confirming!

File: docs/product-manuals/cloud-console/manage-organization/manage-users.md
Comment (christinaausley) [reply docs/product-manuals/cloud-console/manage-organization/manage-users.md:36]: Awesome -- thanks for clarifying.

File: docs/product-manuals/cloud-console/troubleshooting/common-pitfalls.md
Comment (christinaausley) [reply docs/product-manuals/cloud-console/troubleshooting/common-pitfalls.md:21]: How's the following: "Check if your cluster is **Healthy**: A Zeebe cluster may be temporarily unavailable. To check if your cluster is healthy, navigate to the cluster list. Click on the cluster to view its details for a closer view of the cluster status over all products (Zeebe, Operate, Tasklist, Optimize).

File: docs/product-manuals/cloud-console/manage-plan/available-plans.md
Comment (akeller) [inline docs/product-manuals/cloud-console/manage-plan/available-plans.md:14]: @christinaausley can you reach out to Dan Levy about this section? I'd like to get his thoughts on a few things

* Should we change the reference to the GA release date? 
* When we talk about our plans, do we really have 3? Based on the [product page](https://camunda.com/products/cloud/) it looks like you can trial the Professional or Enterprise plans.

**Review by akeller — COMMENTED** [review]
This looks good to me! I left one comment about positioning our plans.

File: docs/product-manuals/cloud-console/manage-plan/available-plans.md
Comment (christinaausley) [reply docs/product-manuals/cloud-console/manage-plan/available-plans.md:14]: @akeller Dan says 1.0 launched in May 2021; 1.1 in July. He also thinks two plans are better suited here. Essentially, two plans with a free trial available. I can adjust these changes on my end.

File: docs/product-manuals/cloud-console/troubleshooting/common-pitfalls.md
Comment (urbanisierung) [reply docs/product-manuals/cloud-console/troubleshooting/common-pitfalls.md]: I like it!

**Comment by christinaausley** [comment]
@urbanisierung Would you like to wait until Schuppi is back before we merge? Definitely no rush, but happy to close this one out if you want to.

**Comment by urbanisierung** [comment]
> @urbanisierung Would you like to wait until Schuppi is back before we merge? Definitely no rush, but happy to close this one out if you want to.

@christinaausley We can merge it as soon as we're ready. From my point of view this can be merged.

**Review by urbanisierung — APPROVED** [review]
/lgtm

thanks!

**Pull request merged**
